devices/rtx/visrtx_device.json

```diff
@@ -45,7 +45,7 @@
       "khr_volume_transfer_function1d",
       "nv_array_cuda",
       "nv_frame_buffers_cuda",
-      $<$<BOOL:$<TARGET_EXISTS:MDL_SDK::MDL_SDK>>:"visrtx_material_mdl",>
```

Comment:
This removal also looks like a mistake?

I think we also should have a json extension defined for this new geometry extension, something like `visrtx_geometry_neural`.

Reply:
The removal is a mistake, and I definitely missed the new extension. Fixed.